integrations/azure-ai-search.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: integration
+name: Azure AI Search
+description: Use Azure AI Search with Haystack
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: https://www.linkedin.com/company/deepset-ai/
+pypi: https://pypi.org/project/azure-ai-search
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/azure-ai-search
+type: Model Provider
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
+logo: /logos/azure-ai.png
+version: Haystack 2.0
+toc: true
+---
+
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Overview
+
+`AzureAIDocumentStore` supports an integration of [Azure AI Search](https://learn.microsoft.com/en-us/azure/search/search-what-is-azure-search) which is an enterprise-ready search and retrieval system with [Haystack](https://haystack.deepset.ai/) by [deepset](https://www.deepset.ai).
+
+This integration allows using search indexes in Azure AI Search as a document store to build RAG-based applications on Azure, with native LLM integrations. To retrieve data from the documentstore, the integration supports three types of retrievers:
+
+1. **Embedding Retrieval**: For vector-based searches.
+2. **BM25 Retrieval**: Semantic retrieval utilizing the BM25 algorithm.
+3. **Hybrid Retrieval**: Combining vector and BM25 retrieval methods for optimal results.
```

Comment:
I would say sometimes - it depends on use case. BM25 is better in handling keyword search - think catalog search where semantic is harder

Reply:
Would you like to add more detail like in which cases Hybrid retrieval is better? Otherwise I'l just keep it simple by describing its a combination of above retrieval methods.

Reply:
No more detailed needed - it was a nit  :-) - I wanted to say that adding a single word can make a statement more true.